src/main/java/org/vanilladb/core/storage/buffer/BufferPoolMgr.java

```diff
@@ -33,7 +34,9 @@
 	private AtomicInteger numAvailable;
 
 	// Optimization: Lock striping
-	private Object[] anchors = new Object[1009];
+	private int stripSize = 1009;

```

Comment:
This would be better to be a `static final` member.

Reply:
Great suggestion.